WordPressAuthenticator/Unified Auth/View Related/2FA/TwoFAViewController.swift

```diff
@@ -384,8 +384,8 @@ private extension TwoFAViewController {
     /// Configure the textfield cell.
     ///
     func configureTextField(_ cell: TextFieldTableViewCell) {
-        cell.configureTextFieldStyle(with: .numericCode,
-                                     and: WordPressAuthenticator.shared.displayStrings.twoFactorCodePlaceholder)
+        cell.configure(withStyle: .numericCode,
```

Comment:
> In this case we already know we're dealing with a style.

Well, not necessarily. Previously, it wasn't immediately clear what the params `with` and `and` were. It was even requested during a previous PR review to clarify the param names. Regarding the style, I've simply moved the word from the method name to the param name to make it more clear. And the method signature is now less wordy since I removed the `TextField` bit from the method name (to me, _that_ was obvious).

Reply:
This is minor, no need to address this if you disagree.

I still think the type is implicitly evident, but it's not something worth holding a PR back on.